modules/dynamic-operator/api/v1alpha1/kuberlogicservice_types.go

```diff
@@ -40,7 +40,10 @@ type KuberLogicServiceSpec struct {
 	// Resources (requests/limits)
 	Limits v1.ResourceList `json:"limits,omitempty"`
 
-	Domain string `json:"domain,omitempty"`
+	// +kubebuilder:validation:Required
```

Comment:
Maybe be better to check the requirements in the plugin side? What the domain should be using in the tests (there we are using postgresql plugin )?

Reply:
Makes sense